templates/quickstart-vpc-for-atlassian-services.yaml

```diff
@@ -191,11 +215,9 @@ Resources:
     Condition: ProvisionBastion
     Properties:
       TemplateURL: !Sub
-        - https://${QSS3BucketName}.${QSS3Region}.amazonaws.com/${QSS3KeyPrefix}templates/quickstart-bastion-for-atlassian-services.yaml
-        - QSS3Region: !If
-            - GovCloudCondition
-            - s3-us-gov-west-1
-            - s3
+        - https://${S3Bucket}.s3.${S3Region}.${AWS::URLSuffix}/${QSS3KeyPrefix}submodules/quickstart-atlassian-services/templates/quickstart-bastion-for-atlassian-services.yaml
```

Comment:
Now that we are passing in the QSS3KeyPrefix to this template as a parameter we can actually source templates from a custom bucket, currently we are just using the default bucket. As a result we need to add `quickstart-atlassian-services` into the path